lib/SystemStatistics.php

```diff
@@ -94,4 +95,25 @@ protected function getMemoryUsage() {
 		];
 	}
 
+	/**
+	 * Get current CPU load average
+	 *
+	 * @return array load average with three values, 1/5/15 minutes average.
+	 */
+	protected function getProcessorUsage() {
+		// get current system load average.
+		$loadavg = sys_getloadavg();
+
+		// check if we got any values back.
+		if (!(is_array($loadavg) && count($loadavg) === 3)) {
+			// either no array or too few array keys.
+			// returning back zeroes to prevent any errors on JS side.
+			$loadavg = [0, 0, 0];
+		}
+
+		return [
+			'loadavg' => $loadavg
```

Comment:
To allow easier extending the information without breaking something on the JS side. To be more extensible for the future. when extending stats with more CPU graph lines someday (e.g. splitting up CPU usage in user, system, kernel or something like that). But I can remove array if you want.